pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go

```diff
@@ -344,7 +344,7 @@ func getMaxVolLimitFromEnv() int {
 		if parsedMaxVols, err := strconv.Atoi(rawMaxVols); err != nil {
 			klog.Errorf("Unable to parse maximum PD volumes value, using default: %v", err)
```

Comment:
@chendave what do you think about adding this to the PR to be `ErrorS`?

I see what you mean about the others not benefitting from it. But while we are making a change here, might as well throw this one in too. Sorry for the back and forth

Reply:
Done